tests/fixtures.ts

```diff
@@ -30,7 +30,7 @@ export class CodeEditors {
     nth = 0,
     formatOutput = true,
   }: {nth?: number; formatOutput?: boolean} = {}) {
-    return await getMonacoContent({locator: this.codeEditors.nth(nth), formatOutput})
+    return await getMonacoContent({locator: this.codeEditors.nth(nth + 1), formatOutput})
```

Comment:
Too smart, is there a reason why you didn't want to default it to 1?

Reply:
Oh now now I got the reasoning behind the change. Makes totally sense

Reply:
I think it is more error-prone. You can call the function with nth set to 0 without knowing you are pointing to the schema code editor.